core/src/main/scala/scalacss/package.scala

```diff
@@ -12,6 +12,10 @@ package object scalacss {
 
   private[scalacss] final val _important = " !important"
 
+  case class UnicodeRange(from: Int, to: Int) {
+    override def toString: String = s"U+${"%x" format from}-${"%x" format to}"
```

Comment:
Better to use: `"U+%x-%x".format(from, to)`. Also either put a newline after `: String` or remove the `: String` if you keep on same line.